### PR TITLE
fix: add translation for toast aria-label

### PR DESCRIPTION
**Pull request by felicia-haggqvist**
Fixes JIRA ticket: [WARP-348](https://nmp-jira.atlassian.net/browse/WARP-348)

- Add translation to aria-label in toast component.

### CODE REVIEW

File: packages/toast/locales/en/messages.po [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: @lingui/cli\n"
+"Language: en\n"
+
+#. Default screenreader message for error in toast component
+#. js-lingui-explicit-id
+#: packages/toast/toast.js:112
+msgid "toast.aria.error"
+msgstr "error"
```
Comment (BalbinaK) [inline packages/toast/locales/en/messages.po:14]: Shouldn't the strings be with Capital letter?

Comment (felicia-haggqvist) [reply packages/toast/locales/en/messages.po:14]: You're right! I'll change it! 😄

File: packages/toast/locales/nb/messages.po [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+#. js-lingui-explicit-id
+#: packages/toast/toast.js:118
+msgid "toast.aria.successful"
+msgstr ""
+
+#. Default screenreader message for warning in toast component
+#. js-lingui-explicit-id
+#: packages/toast/toast.js:106
+msgid "toast.aria.warning"
+msgstr ""
```
Comment (BalbinaK) [inline packages/toast/locales/nb/messages.po:26]: Unless you want to wait with publishing a new version until all translations are in place, you could manually add the translations we had before and ship it without breaking accessibility on finn.no

Comment (felicia-haggqvist) [reply packages/toast/locales/nb/messages.po]: Added Norwegian translation in this commit: [3ac496e](https://github.com/warp-ds/elements/pull/123/commits/3ac496e7c23d825159e03800f13154265689f3ee)

Comment (felicia-haggqvist) [reply packages/toast/locales/nb/messages.po]: But do I also need to manually change the messages.mjo file as well for Norwegian?

Comment (BalbinaK) [reply packages/toast/locales/nb/messages.po]: Once you update .po files, you run `messages:compile` again to see mjs files being updated.

Comment (felicia-haggqvist) [reply packages/toast/locales/nb/messages.po]: Oh right! 😄 Ok! 👍

File: packages/toast/locales/en/messages.po (hunk 1 above)
Comment (felicia-haggqvist) [reply packages/toast/locales/en/messages.po]: Updated messages.mjs in this commit: [c2d0332](c2d0332bc19317303a6b58af6a0a0021bf65a1f3)

**Review by BalbinaK — APPROVED** [review]

**Pull request merged**